src/gui/qgsprovidersublayersdialog.h

```diff
@@ -48,24 +62,48 @@ class QgsProviderSublayerDialogModel : public QgsProviderSublayerModel
 
 };
 
-class QgsProviderSublayersDialog : public QDialog, private Ui::QgsProviderSublayersDialogBase
+/**
+ * \ingroup gui
+ *
+ * \brief Dialog for selecting provider sublayers.
+ *
+ * \since QGIS 3.30
+ */
+class GUI_EXPORT QgsProviderSublayersDialog : public QDialog, private Ui::QgsProviderSublayersDialogBase
 {
     Q_OBJECT
   public:
 
+    /**
+     * Constructor.
+     */
     QgsProviderSublayersDialog( const QString &uri,
                                 const QString &filePath,
                                 const QList< QgsProviderSublayerDetails> initialDetails = QList< QgsProviderSublayerDetails>(),
                                 const QList< QgsMapLayerType > &acceptableTypes = QList< QgsMapLayerType >(),
                                 QWidget *parent SIP_TRANSFERTHIS = nullptr,
                                 Qt::WindowFlags fl = Qt::WindowFlags() );
 
+    /*
+     * Set list of non-layer items.
```

Comment:
If you happen to know, what this refers to, a more verbose version would be appreciated

Reply:
Not directly familiar with the code, but I can try and formulate a better version. But I'm currently at loss why the documentation test still believes that these methods are undocumented?

Reply:
For the record, the model docs explain further what non layer items are. Currently only embedded projects are using these.